ale_linters/swift/swiftlint.vim

```diff
@@ -1,9 +1,9 @@
 " Author: David Mohundro <david@mohundro.com>
 " Description: swiftlint for swift files
 
-call ale#linter#Define('swiftlint', {
+call ale#linter#Define('swift', {
 \   'name': 'swiftlint',
 \   'executable': 'swiftlint',
-\   'command': g:ale#util#stdin_wrapper . ' .swift swiftlint',
+\   'command': g:ale#util#stdin_wrapper . ' .swift swiftlint lint --path',
```

Comment:
Right, I must have missed it when going through the docs. Updated now, appears to work fine as well.